app/models/web_search.rb

```diff
@@ -138,10 +138,15 @@ def module_tag_for_search_engine
 
   def post_process_results(results)
     post_processor = WebResultsPostProcessor.new(@query, @affiliate, results)
-    @normalized_results = post_processor.normalized_results
+    process_data_for_redesign(post_processor)
     post_processor.post_processed_results
   end
 
+  def process_data_for_redesign(post_processor)
+    @normalized_results = post_processor.normalized_results
+    @total_pages = post_processor.total_pages(@total)
```

Comment:
Again - I would make `total_pages` part of the `normalized_results` to allow the front-end code to rely on a single source of data.